wallet/test/utils/cryptography/mnemonics.test.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
```

Comment:
In general I would prefer the tests to be next to the files they are testing instead of the test directory. Not a problem for the PR. (Just a heads up that I might move them)